### DISCUSSION TITLE
Can I enable MSAA in open3d Visualizer?

### DISCUSSION THREAD

**Discussion by yaronaddar** [isl-org/Open3D#6159] [Q&A]
hey, I want to change the step of each jump in size from 1.0 to 0.1, so I did it in renderOption.h:

```cpp
...
 const double POINT_SIZE_MIN = 1.0;
    const double POINT_SIZE_STEP = 0.1; // here was 1.0 so i change it to 0.1
    const double POINT_SIZE_DEFAULT = 5.0;
    ...
```
and afterwords i printed the point size and it changes by 0.1, but the cloud itself wont increase/decrease the pointsize only when it reach to 1.5, 2.6, 3.5, 4.6....

so i think its because I dont have msaa enable? maybe im wrong. can anyone help me with that issue?

**Comment by errissa** [comment]
@yaronaddar Correct - pixel point sizes are integral so you won't see a change until you get to x.5. MSAA may help. With the old Visualizer there is no way to programmatically enable MSAA though you might be able to do so through system settings.

With the new visualizer MSAA is enabled by default and can be controlled programmatically.
